app/src/androidTest/java/org/mozilla/focus/helper/BeforeTestTask.java

```diff
@@ -51,6 +52,7 @@ public void execute() {
         }
 
         Inject.getTabsDatabase(null).tabDao().deleteAllTabs();
+        BookmarksDatabase.getInstance(context).bookmarkDao().deleteAllBookmarks();
```

Comment:
Why is this needed? shouldn't all bookmarks be empty when the app first launched?